build.d/750-geoip-install

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+if [[ "$GEOIP_ACCOUNT_ID" != "" && "$GEOIP_LICENSE_KEY" != "" ]] ; then
+    set -e
+    # see https://dev.maxmind.com/geoip/geoipupdate/
+    echo "Activating GeoIP/GeoLite2 updates"
+    echo "This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com"
+    echo "
+# GeoIP.conf file for \`geoipupdate\` program, for versions >= 3.1.1.
+# Used to update GeoIP databases from https://www.maxmind.com.
+# For more information about this config file, visit the docs at
+# https://dev.maxmind.com/geoip/geoipupdate/.
+
+# \`AccountID\` is from your MaxMind account.
+AccountID ${GEOIP_ACCOUNT_ID}
+
+# Replace YOUR_LICENSE_KEY_HERE with an active license key associated
+# with your MaxMind account.
+LicenseKey ${GEOIP_LICENSE_KEY}
+
+# \`EditionIDs\` is from your MaxMind account.
+EditionIDs GeoLite2-City
+    " > /etc/GeoIP.conf
+    curl --silent -L --output geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb https://github.com/maxmind/geoipupdate/releases/download/v${GEOIP_UPDATER_VERSION}/geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb
+    dpkg -i --force-confold geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb
+    rm geoipupdate_${GEOIP_UPDATER_VERSION}_linux_amd64.deb
+    chown -R root:odoo /usr/share/GeoIP/
+    chmod g+w /usr/share/GeoIP/
+
+    if [[ ! -e /opt/odoo/common/conf.d/60-geoip.conf ]] ; then
+        echo "
+[options]
+# This product includes GeoLite2 data created by MaxMind, available from https://www.maxmind.com
+geoip_database=/usr/share/GeoIP/GeoLite2-City.mmdb
```

Comment:
The idea here was that one can overwrite the 60-geoip.conf in scaffolding and define something different if needed.

Reply:
Actually that would be the default behavior. Any time you write a config that already exists, the generated file always gets the last value.

Reply:
As far as i understood it that only works on variable level. Like a=b in doodba and a=c in scaffolding would set a=c. But if you want to get rid of the variable definition like no a=... statement in the config this is currently not possible to do that in scaffolding if its already defined in doodba. So the idea was that if you do not want deoip_database defined with any value you still could do that by providing an empty (just with `[options]`) 60-geoip.conf in your scaffolding.

Reply:
Well, I think we're not understanding each other.

See, my question was about if odoo supports setting an unexisting file in this setting. It seems to be supported since v8+ (where geoip support started) and [all it would do is to produce a warning](https://github.com/odoo/odoo/blob/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b/addons/website/models/ir_http.py#L96). Other versions behave similarly, and this seems a good behavior, so please just add the raw `conf.d/60-geoip.conf` file (you don't need to create it or not depending on circumstances).

Reply:
I added the raw file, also it seems there is always a default set in the config https://github.com/odoo/odoo/blob/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b/openerp/tools/config.py#L306, so you're right it doesn't matter that we always add 60-geoip.conf.